dev/bots/test.dart

```diff
@@ -975,3 +1009,8 @@ Future<void> _androidGradleTests(String subShard) async {
     await _runDevicelabTest('module_host_with_custom_build_test', env: env);
   }
 }
+
+/// Returns true if `p` logically implies `q`, false otherwise.
+///
+/// If `p` is true, `q` must be true.
+bool _implies(bool p, bool q) => !p || q;
```

Comment:
I don't think this function is carrying its weight here.

Reply:
I think it makes the assertion more readable and makes its intent clearer, but I can remove it if you like.

Reply:
Please remove